english/doug-adams/doug-adams.md

```diff
@@ -0,0 +1,8 @@
+Where did all the dolphins go?
+
+Dolphins are smart creatures, that much we know. However, it had never dawned upon us just how smart they actually were. The farewell message that they had sent prior to their departure was both enigmatic and ridiculous at the same time. _So long and thanks for all the fish_? Who would have thought... Who would have known?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.